.hassio-addons.yml

```
@@ -73,6 +73,10 @@ addons:
     repository: hassio-addons/addon-lutron-cert
     target: lutron-cert
     image: hassioaddons/lutron-cert-{arch}
+  magicmirror:
+    repository: hassio-addons/addon-magicmirror
+    target: magicmirror
+    image: hassioaddons/magicmirror
```

We should no longer rely on Docker manifests.
Docker has a bug with ARMv6 / ARMv7 which causes ARMv6 machines to download the ARMv7 image.

```suggestion
    image: hassioaddons/magicmirror-{arch}
```